dist/migration/class-install-genesis-blocks-api.php

```diff
@@ -0,0 +1,235 @@
+<?php
+/**
+ * Migration REST API endpoints.
+ *
+ * @package   Atomic_Blocks
+ * @copyright Copyright(c) 2020, Block Lab
+ * @license http://opensource.org/licenses/GPL-2.0 GNU General Public License, version 2 (GPL-2.0)
+ */
+
+namespace Atomic_Blocks\Admin\Migration;
```

Comment:
Small thing. We've used `AtomicBlocks` as the namespace in other parts of the plugin. Would be good to keep it consistent here.